src/libgpu/latte/latte_enum_sq.h

```diff
@@ -247,6 +247,7 @@ ENUM_BEG(SQ_SEL, uint32_t)
    ENUM_VALUE(SEL_W,                      3)
    ENUM_VALUE(SEL_0,                      4)
    ENUM_VALUE(SEL_1,                      5)
+   ENUM_VALUE(SEL_BIAS,                      6)
```

Comment:
Yes, that is the easiest option if you're new to git.
They can all be squashed down to 1 commit later, so no problem there.
You're welcome :)